examples/minimal/client/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@examples/minimal-client",
```

Comment:
Let's add `"private": true` to these too

Reply:
fixed